Packs/CrowdStrikeFalcon/ReleaseNotes/2_1_10.md

```diff
@@ -0,0 +1,272 @@
+
+Ensured incident field only appears in Cortex XSOAR.
+
+##### CrowdStrike Falcon Firmware Build Fingerprint
+
+Ensured incident field only appears in Cortex XSOAR.
+
+##### CrowdStrike Falcon Alleged File Type
+
+Ensured incident field only appears in Cortex XSOAR.
+
+##### CrowdStrike Falcon Response Elements
+
+Ensured incident field only appears in Cortex XSOAR.
+
+##### CrowdStrike Falcon Vertex Type
+
+Ensured incident field only appears in Cortex XSOAR.
+
+##### Behaviour Scenario
+
+Ensured incident field only appears in Cortex XSOAR.
+
+##### Grand Parent Process
+
+Ensured incident field only appears in Cortex XSOAR.
+
+##### CrowdStrike Falcon Macros IOC Source
+
+Ensured incident field only appears in Cortex XSOAR.
+
+
+#### Incident Types
+
+##### CrowdStrike Falcon Detection
+
+Ensured incident type only appears in Cortex XSOAR.
+
+##### CrowdStrike Falcon Incident
+
+Ensured incident type only appears in Cortex XSOAR.
+
+##### CrowdStrike Falcon On-Demand Scans Detection
+
+Ensured incident type only appears in Cortex XSOAR.
+
+##### CrowdStrike Falcon Mobile Detection
+
+Ensured incident type only appears in Cortex XSOAR.
+
+##### CrowdStrike Falcon IOM Event
+
+Ensured incident type only appears in Cortex XSOAR.
+
+##### CrowdStrike Falcon OFP Detection
+
+Ensured incident type only appears in Cortex XSOAR.
+
+##### CrowdStrike Falcon IDP Detection
+
+Ensured incident type only appears in Cortex XSOAR.
+
+##### CrowdStrike Falcon IOA Event
+
+Ensured incident type only appears in Cortex XSOAR.
+
+
+#### Mappers
+
+##### Legacy CrowdStrike Falcon-Mapper
+
+Ensured mapper only appears in Cortex XSOAR.
+
+##### CrowdStrike Falcon Mapper
+
+Ensured mapper only appears in Cortex XSOAR.
+
+##### CrowdStrike Falcon - Outgoing Mapper
+
+Ensured mapper only appears in Cortex XSOAR.
+
+
+#### Playbooks
+
+##### CrowdStrike Falcon - Search Endpoints By Hash
+
+Ensured playbook only appears in Cortex XSOAR.
+
+##### CrowdStrike Falcon - T1059 - Command and Scripting Interpreter
+
+Ensured playbook only appears in Cortex XSOAR.
+
+##### CrowdStrike Falcon - Get Detections by Incident
+
+Ensured playbook only appears in Cortex XSOAR.
+
+##### Crowdstrike Falcon - Unisolate Endpoint
+
+Ensured playbook only appears in Cortex XSOAR.
+
+##### CrowdStrike Falcon - Get Endpoint Forensics Data
+
+Ensured playbook only appears in Cortex XSOAR.
+
+##### CrowdStrike Falcon - Search Endpoints By Indicators
+
+Ensured playbook only appears in Cortex XSOAR.
+
+##### Crowdstrike Falcon - Isolate Endpoint
+
+Ensured playbook only appears in Cortex XSOAR.
+
+##### CrowdStrike Falcon - Retrieve File
+
+Ensured playbook only appears in Cortex XSOAR.
+
+##### CrowdStrike Falcon - Block File
+
+Ensured playbook only appears in Cortex XSOAR.
+
+##### CrowdStrike Falcon Malware - Verify Containment Actions
+
+Ensured playbook only appears in Cortex XSOAR.
+
+
+#### Scripts
+
+##### ConvertEnrichmentsToTable
+
+Ensured script only appears in Cortex XSOAR.
+-->
```

Comment:
No need for all of it to be a comment. It is irrelevant to customers.

Reply:
It is in a comment for the following reasons:
* To document this in the code without being visible in the official release notes 
* To pass the SDK release notes validation (if certain titles do not appear, the release notes are considered invalid)